alert-policies/lighttpd/NumberOfRequests.yml

```diff
@@ -1,10 +1,10 @@
-name: Average Number of Requests
+name: Number of Requests (%)
 
 description: |+
-  This alert is triggered when the average number of requests exceeds 4000 for 2 minutes.
+  This alert is triggered when the number of requests exceeds 80% for 2 minutes.
 type: STATIC
 nrql:
-  query: "FROM lighttpdStatusSample2 SELECT latest(requestsAvg) As 'Average Number of Requests'"
+  query: "SELECT latest(RequestAverage5s)/latest(RequestsTotal) * 100 AS 'Number of Requests (%)' FROM LighttpdStatusSample  where RequestAverage5s > 0"
```

Comment:
query: "SELECT (latest(RequestAverage5s) * 100)/latest(RequestsTotal) AS 'Number of Requests (%)' FROM LighttpdStatusSample  where RequestAverage5s > 0"